src/System.Text.RegularExpressions/tests/Regex.Match.Tests.cs

```diff
@@ -28,6 +30,16 @@ public static IEnumerable<object[]> Match_Basic_TestData()
             // Using *, +, ?, {}: Actual - "a+\\.?b*\\.?c{2}"
             yield return new object[] { @"a+\.?b*\.+c{2}", "ab.cc", RegexOptions.None, 0, 5, true, "ab.cc" };
 
+            // Using long loop prefix
+            yield return new object[] { @"a{10}", new string('a', 10), RegexOptions.None, 0, 10, true, new string('a', 10) };
+            yield return new object[] { @"a{100}", new string('a', 100), RegexOptions.None, 0, 100, true, new string('a', 100) };
+
+            yield return new object[] { @"a{10}b", new string('a', 10) + "bc", RegexOptions.None, 0, 12, true, new string('a', 10) + "b" };
+            yield return new object[] { @"a{100}b", new string('a', 100) + "bc", RegexOptions.None, 0, 102, true, new string('a', 100) + "b" };
+
+            yield return new object[] { @"a{11}b", new string('a', 10) + "bc", RegexOptions.None, 0, 12, false, string.Empty };
+            yield return new object[] { @"a{101}b", new string('a', 100) + "bc", RegexOptions.None, 0, 102, false, string.Empty };
```

Comment:
can we have tests with quantifiers like {a,b} and {a,}

Reply:
Will do. I discovered that we do not support `{,n}` for some reason. Unfortunately it is treated as a literal (not error) so we probably can't switch it on. Workaround is `{0,n}`.

Reply:
Yeah, i thought that was odd myself.  But it's not in the Regex reference, so it's at least expected as per our docs.